### PR TITLE
[[ TSV ]] Use .tsv extension for tab delimited files

### PR DESCRIPTION
**Pull request by bwmilby**
Renamed tab separated text files with .tsv extension (property definitions/standalone settings)
Adjusted code (IDE Library) to load updated files
Formatted all files to a standard number of tabs to support Git formatting of tables
Moved field headings to the top of the file to support Git formatting of tables

### CODE REVIEW

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 17076c4

Thanks @bwmilby ! I'm sure this will make it much easier to read these files. I'm reviewing OK but will check with @livecodeali that he is happy for the headings to be above those lines at the top before merging. It does look a little odd but at least if you search you get the correct titles.

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 17076c47a6d6fd833b6d05022140480d55116463

File: Toolset/resources/supporting_files/standalone_settings/standaloneSettings.tsv [hunk 1]
```diff
@@ -1,131 +1,131 @@
...
+									
 inclusions		General	com.livecode.pi.radiogroup	true	false	Inclusions	search	Search for required inclusions when saving the application|search,Select inclusions for the standalone application|select	Inclusions
 askDialog	Ask Dialog	General	com.livecode.pi.boolean	true	false	Inclusion Options			Inclusions
 answerDialog	Answer Dialog	General	com.livecode.pi.boolean	true	false	Inclusion Options			Inclusions
 cursors	Cursors	General	com.livecode.pi.boolean	true	false	Inclusion Options			Inclusions
 revolutionprintdialogs	Print Dialogs	General	com.livecode.pi.boolean	true	false	Inclusion Options			Inclusions
 Brushes	Brushes	General	com.livecode.pi.boolean	true	false	Inclusion Options			Inclusions
-pdfPrinter	PDF Printer	General	com.livecode.pi.boolean	true	false	Inclusion Options		Inclusions
+pdfPrinter	PDF Printer	General	com.livecode.pi.boolean	true	false	Inclusion Options		Inclusions	
 scriptLibraries	Script Libraries	General	com.livecode.pi.set	true	false			execute:dispatch function "revSBLibraryList" to stack "revSBLibrary";get the result;sort it	Inclusions
```
Comment (bwmilby) [inline Toolset/resources/supporting_files/standalone_settings/standaloneSettings.tsv]: Formatter doesn't like this line (13) (complains about illegal quoting)

**Comment by livecodeali** [comment]
I'm fine with that, it was just an arbitrary marker really. The code in the IDE checks for `-----` at the beginning of the line to start reading property info, (as I'm sure @bwmilby figured out) so that's the only requirement without changing the IDE code.

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 4a4ff25

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 4a4ff251043704efb819bf0570e3f81e83780f71

**Pull request merged**